api-client/src/errors/NotFoundError.ts

```diff
@@ -30,11 +30,13 @@ class NotFoundError extends Error {
       Error.captureStackTrace(this, NotFoundError)
     }
 
+    // default values are provided for cases where parameters are unknown
+    // e.g. when throwing NotFoundError from an endpoint
```

Comment:
Same as above, comments should only be necessary if workarounds, complicated logic or similar need to be explained